libs/ui/src/screens/unlock-app/unlock-app.tsx

```diff
@@ -47,17 +50,29 @@ export const UnlockApp: FC = () => {
 
   useEffect(() => void (!isLocked && goBack()), [isLocked]);
 
-  const password = watch('password');
+  useEffect(() => {
+    if (isWeb) {
+      setFocus('password');
+    }
+  }, []);
 
-  const { commonRules } = useValidateForm();
+  const password = watch('password');
 
   const onUnlock = () => unlock(password);
   const onResetWallet = () => navigate(ScreensEnum.SettingsResetWalletConfirm);
 
-  const isSubmitDisabled = (Object.keys(errors).length > 0 && isSubmitted) || unlockError !== '';
+  const isSubmitDisabled = (Object.keys(errors).length > 0 && isSubmitted) || isDefined(unlockError);
+
+  const onEnterPress = isWeb
+    ? (event: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
+        if (event.nativeEvent.key === 'Enter') {
+          onUnlock();
+        }
+      }
+    : undefined;
```

Comment:
short return plz  
do we need `isWeb` check here? maybe on mobile we need the same behaviour

Reply:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/35599200/209343390-46547254-776b-4d47-a153-f78b3712e71f.png">
I will leave the current behavior for now, because on mobile there is no explicit enter